app/assets/stylesheets/components/awaiting_feedback_vacancies.scss

```diff
@@ -0,0 +1,3 @@
+.job-title-table-header {
```

Comment:
Minor comment, do we want to use a more specific class name.  This only applies to the awaiting feedback table headings.

Reply:
I only used this with the `job-title` heading which is why I named it specific.
Didn't want to imply that this would be useful for all the awaiting feedback table headings.
Could change this if this styling is used in more than one place.